src/Card/Card.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import TinderCard from "react-tinder-card";
+
+export default function Card() {
+  const onSwipe = (direction) => {
+    // TODO: Replace with a call to the database
+    console.log(`you swiped: ${direction}`);
+  };
+
+  return (
+    <TinderCard onSwipe={onSwipe} preventSwipe={["up", "down"]}>
```

Comment:
Variables like `up` or `down` could be changed to constants/enums

Reply:
After looking at the Airbnb React/JSX Style Guide and React Tinder Card's documentation, I think it is fine to write it like this.